uint8 gear
uint16 speed
uint16 steer
uint16 brake
bool en_rst